Fixes this error:

../../src.x11/xcmds.c: In function ‘FunChangeList’:
../../src.x11/xcmds.c:715:36: error: passing argument 2 of ‘XawListChange’ from incompatible pointer type [-Wincompatible-pointer-types]
  715 |     XawListChange( arg->sel->list, (const char **)text, 0, 0, True );
      |                                    ^~~~~~~~~~~~~~~~~~~
      |                                    |
      |                                    const char **
In file included from ../../src.x11/utils.h:120,
                 from ../../src.x11/xcmds.c:9:
/usr/include/X11/Xaw/List.h:170:26: note: expected ‘char **’ but argument is of type ‘const char **’
  170 |  String                 *list,
      |  ~~~~~~~~~~~~~~~~~~~~~~~~^~~~

--- src.x11/xcmds.c.orig	2024-02-07 16:26:14.000000000 -0700
+++ src.x11/xcmds.c	2024-03-15 16:59:24.433308375 -0600
@@ -712,7 +712,7 @@ static Boolean FunChangeList (
     text[i] = 0;
 
     /* change list */
-    XawListChange( arg->sel->list, (const char **)text, 0, 0, True );
+    XawListChange( arg->sel->list, text, 0, 0, True );
 
     /* clear old text */
     for ( i = 0;  arg->sel->text[i];  i++ )
